frontend/src/pages/notebookController/screens/admin/NotebookAdminControl.tsx

```diff
@@ -89,11 +90,17 @@ const NotebookAdminControl: React.FC = () => {
                 <Tbody>
                   {users.slice(perPage * pageIndex, perPage * pageIndex + perPage).map((user) => (
                     <Tr key={user.name}>
-                      {columns.map((column) => (
-                        <Td key={column.field} dataLabel={column.field}>
-                          <UserTableCellTransform user={user} userProperty={column.field} />
-                        </Td>
-                      ))}
+                      {columns.map((column) =>
+                        column.field === 'actions' ? (
+                          <Td key={column.field} isActionCell>
+                            <NotebookActions data={user[column.field]} />
+                          </Td>
+                        ) : (
+                          <Td key={column.field} dataLabel={column.field}>
+                            <UserTableCellTransform user={user} userProperty={column.field} />
+                          </Td>
```

Comment:
Why not:
```
<Td key={column.field} dataLabel={column.field} isActionCell={column.field === 'actions'}>
  <UserTableCellTransform user={user} userProperty={column.field} />
</Td>
```
And put the cell renderer inside the `UserTableCellTransform`?

Reply:
good suggestion!